### ISSUE TITLE
Terms of service

### ISSUE THREAD

**Issue by bmackinney** [pantheon-learning/questions#7]
Do we need a terms of service?
What should it include?

**Comment by sukottokun** [comment]
Yes, let's discuss, would like to get this done in Q4. Also: code/site ownership, live site downtime in regards to emergency support.

**Comment by rachelwhitton** [comment]
Should we include a privacy statement in the TOS? 
Would the existing apply? https://pantheon.io/privacy

**Comment by sukottokun** [comment]
Yeah, maybe grouping all of it together would be good.
